deploy/pulumi/Pulumi.default.yaml

```diff
@@ -0,0 +1,57 @@
+config:
+
+  # Cloud provider specific configuration ------------------------------------------------------------------------------
+
+  # Choose a provider to use, one of azure, gcloud, aws
+  prefect-server:provider: gcp
+
+  # Azure ----------------------------------------------------------
+
+  azure:environment: public
+  azure:location: WestUS
+  prefect-server-azure:k8s-node-type: Standard_B1ms
```

Comment:
I would change to Standard_B2ms (2vcpu, 4 GiB)

Reply:
👍